index.js

```diff
@@ -8,12 +8,19 @@ const cheerio = require("cheerio");
 
 require("colors");
 
-
+const GARFIELD = (process.argv.includes("--garfield") || process.argv.includes("--garfield-mode"));
 const METRIC = process.argv.includes("--metric");
 const GRAMS_PER_OZ = 28.3495;
 const SFSPCA_BASE = "https://www.sfspca.org"
 const ADOPTION_PAGE = `${SFSPCA_BASE}/adoptions/cats`;
 
+if(GARFIELD){
+  if(new Date().getDay() == 1){
```

Comment:
style nit - can you add spaces around after `if` and before `{` on this line and the line above?